examples/index.html

```diff
@@ -55,6 +55,8 @@ <h3 class="text-muted">appmap-d3</h3>
         }
       });
       diagram.render(window.componentDiagramModel);
+      // for easy access and debug from console
+      window.diagram = diagram;
```

Comment:
Is this meant to be here?

Reply:
> I don't understand the removal of `package-lock.json`, can you explain more? It seems like a detrimental change as we can no longer guarantee our dependency versions are in sync.
Not one big open source package has package-lock committed. It's not included in the distribution either. By having package lock in repo when you (re)build your package for distribution you have your old dependencies but users(when installing package) will have newer dependencies. That might lead to unnoticed bugs in user's environments. So as I said, that's why we commit package-lock to product, but no to libraries.

Reply:
Yes, I left `window.diagram = diagram`. It's harmless but very convenient to both me and people who are willing to try out example.

Reply:
I'm not sure this should be, I suppose this line was added by Boris to discover component's methods and fields.